tests/partitioninfo.spec.coffee

```diff
@@ -25,3 +30,55 @@ describe 'Partitioninfo:', ->
 				m.chai.expect(promise).to.eventually.become
 					offset: 4194304
 					size: 20971520
+
+	describe '.getPartitions()', ->
+
+		describe 'given a valid image file', ->
+
+			beforeEach ->
+				@bootRecordReadStub = m.sinon.stub(bootRecord, 'read')
+				@bootRecordReadStub.withArgs('mbr/rpi', 0).returns(Promise.resolve(rpiMBR))
+				@bootRecordReadStub.withArgs('mbr/rpi', 394264576).returns(Promise.resolve(bbbMBR))
```

Comment:
I don't understand this very much. So if we attempt to read the MBR from the raspberry pi image from offset 0 we get the raspberry MBR, but when reading the same raspberry pi MBR from offset 394264576 we get the beagle bone black MBR?